docs/osquery/osquery.asciidoc

```diff
@@ -0,0 +1,394 @@
+
+ * Run live queries with one or more agents
+ * View a history of past queries and their results
+ * Schedule queries to capture changes to OS state over time
+ * Save queries and build a library of queries for specific use cases
+
+The UI is powered by the Osquery Manager integration.
+For information on how to set up Osquery Manager, refer to <<manage-osquery-integrations>>.
+
+[float]
+== Required privileges
+
+To use *Osquery Manager*, you must be assigned to a role with the following privileges:
+
+* `Read` privileges for the `logs-osquery_manager.result*` index.
+* {kib} privileges for **Osquery Manager**. The `All` privilege
+enables you to run, schedule, and save queries. `Read` enables you to
+view live and scheduled query results, but you can run live queries or edit anything.
+
+[float]
+[[osquery-run-query]]
+==  Run live queries
+
+Run a query against one or more agents or policies
+and view the results. This is useful for testing your query and
+troubleshooting problems.
+
+. Open the main menu, and then click *Osquery*.
+. In the *Live queries* view, click **New live query**.
+. Select one or more agents or groups to query. Start typing in the search field,
+and you'll get suggestions for agents by name, ID, platform, and policy.
+. Enter a query or select a query from your saved queries.
++
+[role="screenshot"]
+image::images/enter-query.png[Select saved query dropdown name showing query name and description]
+. Click **Submit**.
+. Review the results in a table, or navigate to *Discover* to dive deeper into the response
+or to *Lens* to build visualizations.
+. To view more information about the request, for example, if there are any failures, open the *Status* tab.
+. When your query captures the data you want, click *Save for later* and define the ID,
+description, and other
+<<osquery-manage-query,details>>.
+. To view a history of the past queries you have run, Open the *Live queries history*.
+.. To replay a query, click image:images/play-icon.png[Right-pointing triangle].
+.. To view the query <<osquery-results,results>> and <<osquery-status,status>>,
+click image:images/table-icon.png[Table icon].
++
+[role="screenshot"]
+image::images/live-query-check-results.png[Results of OSquery]
+
+
+[float]
+[[osquery-schedule-query]]
+== Schedule queries
+
+Group and schedule queries to run on a specified interval, in seconds.
+For example, you might create a group that checks
+for IT compliance-type issues and
+another group that monitors for evidence of malware.
+
+. Open the **Scheduled query groups** tab.
+
+. Click a group name to view its details.
++
+Details include the last time it ran, how many results were returned, and the number of agents it ran against.
+If there are errors, expand row to view the details.
++
+[role="screenshot"]
+image::images/scheduled-query-group.png[Shos last results last time it ran, how many results returned, number of agents it ran against,  Is
```

Comment:
Can you confirm this is the correct image file name? I'm getting an error when I try to build this locally so I'm not sure if it's displaying as expected, but I just noticed that there's an image in the `osquery/images` folder named `scheduled-query-groupds.png` but not one named `scheduled-query-group.png`. The `scheduled-query-groupds.png` looks like the correct image.